packages/nextjs/src/utils/serverIntegrations.ts

```diff
@@ -0,0 +1,60 @@
+import { RewriteFrames } from '@sentry/integrations';
+import { Integration } from '@sentry/types';
+
+const SOURCEMAP_FILENAME_REGEX = /^.*\/.next\//;
+
+/** Default RewriteFrames integration to match filenames in Sentry. */
+export const REWRITE_FRAMES_INTEGRATION = new RewriteFrames({
+  iteratee: frame => {
+    try {
+      if (frame.filename) {
+        frame.filename = frame.filename.replace(SOURCEMAP_FILENAME_REGEX, 'app:///_next/');
+      }
+    } catch {
```

Comment:
Also `catch (_oO) {` as no-arguments catch clauses are not supported in a lot of environments (although TS should be fine transpiling it)